### PR TITLE
Configureable user-agent

### PR DESCRIPTION
**Pull request by tot-ra**
Passes **userAgent** option down the chain Client → Collection → RestHandler → Rest for every network request for this issue: https://github.com/pipedrive/client-nodejs/issues/66

### CODE REVIEW

File: lib/restHandlers.js [hunk 1]
```diff
@@ -80,149 +80,206 @@
...
 	// GET /items
-	RestHandlers.prototype.listItems = function(object, params, callback) {
+	RestHandlers.prototype.listItems = function (object, params, callback) {
 		var self = this,
-			paramsToSupply = _.extend({}, _.isObject(params) ? params : {}, this.options.apiToken ? { api_token: this.options.apiToken } : {}),
-			dataObject = object == 'authorizations' ? { multipart: false, data: paramsToSupply } : { query: qs.stringify(paramsToSupply) },
-			req = rest[object == 'authorizations' ? 'post' : 'get'](apiUrl(object, this.options, false), dataObject);
-
-		req.on('complete', function(data, res) {
+			paramsToSupply = _.extend({}, _.isObject(params) ? params : {}, this.options.apiToken ? {api_token: this.options.apiToken} : {}),
```
Comment (ziimk) [inline lib/restHandlers.js]: Please check the (auto)-formatting in all modified files. Looks like there's a lot of unnecessary changes, like removing spaces, for example `{api_token: this.options.apiToken}`

Comment (tot-ra) [reply lib/restHandlers.js]: Thanks, it was wrong formatting configuration in IDE indeed

**Comment by tot-ra** [comment]
closing, since no action was taken for almost a year
